registry/storage/garbagecollect.go

```diff
@@ -60,26 +60,37 @@ func MarkAndSweep(ctx context.Context, storageDriver driver.StorageDriver, regis
 			return fmt.Errorf("unable to convert ManifestService into ManifestEnumerator")
 		}
 
+		unmarkedManifests := make(map[digest.Digest]struct{})
```

Comment:
Used to keep track of the set of manifests within this repository that are unreachable.